src/pages/docs/creators/python/guides/index.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Golem Python Guides
+title: Golem Python Guides
+type: Introduction
+---
+
+Our Services API provides an abstraction over Golem low-level APIs, which is aimed at making the building of service-oriented applications straightforward for a developer. The abstraction is based on a logical concept of a Service, in other words, an entity that implements the logic of a service application, and which, from Requestor's perspective, follows a certain sequence of states:
```

Comment:
plus, the article looks like a general introduction and yet the intro here suggests it's targeted at the Services API... ... so, if it's a general intro, maybe let's just remove this paragraph from here and move it to the next article (service-model.md)